spec/latest/json-ld-api/index.html

```diff
@@ -984,11 +985,23 @@
               <em>container</em>.</li>
           </ol>
         </li>
+        <li>If <em>value</em> contains the key <code>@context</code>:
+          <ol class="algorithm">
+            <li>Initialize <em>context</em> to the value associated with the
+              <code>@context</code> key, which is treated as a <a>local context</a>.</li>
+            <li>Invoke the <a href="#context-processing-algorithm">Context Processing algorithm</a>
+              using the <a>active context</a> and <em>context</em> as <a>local context</a>.
+              If any error is detected, an
+              <a data-link-for="JsonLdErrorCode">invalid scoped context</a> errors
```

Comment:
IMO, we go overboard on errors as it is. This tells the author that the problem is with a scoped context, rather than with the context in general, so it seems more useful this way to me.